composite/crypto-volatility-index/src/config.ts

```diff
@@ -0,0 +1,11 @@
+import { util } from '@chainlink/ea-bootstrap'
+
+export const CMC_API_KEY = 'CMC_API_KEY'
```

Comment:
This ENV var name seems to differ from the one in the README.

If the adapter will only use at most one data provider that requires an API key, can we just rename this `API_KEY` (instead of CMC specifically)?

Reply:
the adapter is using API_KEY, removed this config as it wasn't in use anymore